lms/djangoapps/certificates/views/webview.py

```diff
@@ -104,48 +95,7 @@ def _update_certificate_context(context, course, user, user_certificate):
         context['organization_short_name'] = partner_short_name
         context['organization_logo'] = organization.get('logo', None)
 
-    context['username'] = user.username
-    context['course_mode'] = user_certificate.mode
-    context['accomplishment_user_id'] = user.id
-    context['accomplishment_copy_name'] = user_fullname
-    context['accomplishment_copy_username'] = user.username
     context['accomplishment_copy_course_org'] = partner_short_name
-    course_title_from_cert = context['certificate_data'].get('course_title', '')
-    accomplishment_copy_course_name = course_title_from_cert if course_title_from_cert else course.display_name
-    context['accomplishment_copy_course_name'] = accomplishment_copy_course_name
-    share_settings = getattr(settings, 'SOCIAL_SHARING_SETTINGS', {})
-    context['facebook_share_enabled'] = share_settings.get('CERTIFICATE_FACEBOOK', False)
-    context['facebook_app_id'] = getattr(settings, "FACEBOOK_APP_ID", None)
-    context['facebook_share_text'] = share_settings.get(
-        'CERTIFICATE_FACEBOOK_TEXT',
-        _("I completed the {course_title} course on {platform_name}.").format(
-            course_title=accomplishment_copy_course_name,
-            platform_name=platform_name
-        )
-    )
-    context['twitter_share_enabled'] = share_settings.get('CERTIFICATE_TWITTER', False)
-    context['twitter_share_text'] = share_settings.get(
-        'CERTIFICATE_TWITTER_TEXT',
-        _("I completed a course on {platform_name}. Take a look at my certificate.").format(
-            platform_name=platform_name
-        )
-    )
-
-    course_number = course.display_coursenumber if course.display_coursenumber else course.number
-    context['course_number'] = course_number
-    try:
-        badge = BadgeAssertion.objects.get(user=user, course_id=course.location.course_key)
-    except BadgeAssertion.DoesNotExist:
-        badge = None
-    context['badge'] = badge
-
-    # Override the defaults with any mode-specific static values
-    context['certificate_id_number'] = user_certificate.verify_uuid
-    context['certificate_verify_url'] = "{prefix}{uuid}{suffix}".format(
-        prefix=context.get('certificate_verify_url_prefix'),
-        uuid=user_certificate.verify_uuid,
-        suffix=context.get('certificate_verify_url_suffix')
-    )
 
     # Translators:  The format of the date includes the full name of the month
     context['certificate_date_issued'] = _('{month} {day}, {year}').format(
```

Comment:
I think `certificate_date_issued` is related to certificate and that's why it is in `_update_certificate_context`